packages/redis/src/index.d.ts

```diff
@@ -10,7 +10,7 @@ declare class KeyvRedis extends EventEmitter implements Store<string | undefined
 	constructor(options?: KeyvRedis.Options | Redis | Cluster);
 	constructor(uri: string | Redis | Cluster, options?: KeyvRedis.Options);
 	get(key: string): Promise<string | undefined>;
-	getMany(keys: string[]): Promise<string[] | undefined>;
+	getMany(keys: string[]): Promise<string[]>;
```

Comment:
@dylanseago - have you validated this is correct as I believe that in some cases it can return an undefined.

Reply:
@jaredwray yes, as per the implementation, it uses `Array.every` on the `rows` returned from redis. So if it were possible for `undefined` to be returned here then this would be throwing a `TypeError` when trying to call on `undefined`.
https://github.com/jaredwray/keyv/blob/main/packages/redis/src/index.js#L38

Although, it appears to be checking if any individual _values_ are `null`. So it may be more correct to return `Promise<Array<string | null>>` instead